virttest/qemu_devices/qcontainer.py

```diff
@@ -2110,6 +2110,8 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
                 protocol_cls = qdevices.QBlockdevProtocolFTPS
             elif filename.startswith('ftp:'):
                 protocol_cls = qdevices.QBlockdevProtocolFTP
+            elif filename.startswith('vdpa:'):
```

Comment:
It seems that the concept is a bit inconsistent, we have introduced storage
_type, why not use this value to deduce?

Reply:
one more question is about naming "storage_type" , I think it is not only about storage type, but also the protocol type, image_protocol is better? consider the same source it could be used different driver or protocol , example: QSD export , it could be used as device vhost-user-blk or driver virtio-blk-vhost-user.  /home/abc it can be used as file protocol or io_uring protocol